manual/softlayer-sru-20.2.45.txt

```diff
@@ -0,0 +1,1161 @@
++ SRU_VARS=./sru-scripts/sru-vars.template
```

Comment:
Is there a reason to not attach the scrip that generated the output here ?

Reply:
+1 added the appropriate script header